runtime/common/CMakeLists.txt

```diff
@@ -105,8 +105,9 @@ install(EXPORT cudaq-common-targets
 get_property(dialect_libs GLOBAL PROPERTY MLIR_DIALECT_LIBS)
 get_property(conversion_libs GLOBAL PROPERTY MLIR_CONVERSION_LIBS)
 
-add_library(cudaq-mlir-runtime SHARED RuntimeMLIR.cpp)
+add_library(cudaq-mlir-runtime SHARED RuntimeMLIR.cpp JIT.cpp)
```

Comment:
Not sure, but wondering if this has implications on LLVM static initialization issue since the `jit_util` depended on LLVM libraries.

Reply:
I think cudaq-mlir-runtime depends on the same libraries, so hopefully it isn't any worse than it already was? (But I don't know a good way to test it, so I'm open to other ideas, too.)

Reply:
@amccaskey can confirm, but I think this should not be any worse than it currently is.

Reply:
Agreed (from the build experiments of past two days).